app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/manager/AccessibilityFieldManager.kt

```diff
@@ -0,0 +1,30 @@
+package com.x8bit.bitwarden.data.autofill.accessibility.manager
+
+import android.net.Uri
+import android.view.accessibility.AccessibilityNodeInfo
+
+/**
+ * A manager for finding fields that match particular characteristics.
+ */
+interface AccessibilityFieldManager {
+    /**
+     * A helper function for retrieving the appropriate nodes based on the given [predicate].
+     *
+     * This function is recursive but will stop recurring if the [recursionDepth] reaches more that
+     * the [MAX_RECURSION_DEPTH].
+     */
+    fun findAccessibilityNodeInfos(
```

Comment:
⛏️ how would you feel about `findAccessibilityNodeInfoList` vs the `s` suffix? I am couldn't figure a way to search if this is a convention already in the codebase but I have adopted an aversion to the `getFoos():Collection<Foo>` naming. Again treating like a court case, if there is precedent just ignore.

Reply:
I'm on board with the "list" suffix.

Reply:
Updated